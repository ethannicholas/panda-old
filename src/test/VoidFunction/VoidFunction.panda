function x() {
}

Console.writeLine(x())