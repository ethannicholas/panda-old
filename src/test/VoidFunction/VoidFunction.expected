error: VoidFunction.panda:1:1: functions must have a return type
1 error
